client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -341,14 +352,23 @@ class ActivityLogTasklist extends Component {
 				</div>
 				{ // Show if plugin update didn't start, is still running or errored,
 				// but hide plugin if it was updated successfully.
-				pluginsToUpdate.map( plugin => (
+				itemsToUpdate.map( item => (
 					<ActivityLogTaskUpdate
-						key={ plugin.id }
-						plugin={ plugin }
-						goToPlugin={ this.goToPlugin }
-						enqueuePlugin={ this.enqueuePlugin }
-						dismissPlugin={ this.dismissPlugins }
-						disable={ isPluginEnqueued( plugin.slug, queuedPlugins ) }
+						key={ item.id }
+						toUpdate={ item }
+						name={ item.name }
+						slug={ item.slug }
+						version={ item.version }
+						type={ item.type }
+						updateType={
+							'plugin' === item.type
+								? translate( 'Plugin update available' )
+								: translate( 'Theme update available' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Theme update available', 'Theme updates available' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).